src/openrct2/actions/BannerPlaceAction.h

```diff
@@ -11,18 +11,12 @@
 
 #include "GameAction.h"
 
-class BannerPlaceActionResult final : public GameActions::Result
+struct BannerPlaceActionResult
```

Comment:
This can be moved out of the header file.

Reply:
Move it where? Its needed in BannerPlaceAction.cpp to assign the data, and callers need to know the type as well.

Reply:
Nevermind, I thought this was only used in the .cpp file.